gensim/models/ldamodel.py

```diff
@@ -182,6 +187,366 @@ def load(cls, fname, *args, **kwargs):
         return result
 # endclass LdaState
 
+# DD
+# wrapper to just return lda.state after performing lda
+from distributed import local_client
+
+
+class DaskWorker(object):
+    #from enum import Enum
+    #class MsgType(Enum):
+    RESET = 'RESET' 
+    EVAL = 'EVAL'
+    GETSTATE ='GETSTATE'
+    DONE = 'DONE'
+
+    
+    def __init__(self, state,inqueue,outqueue,myid, id2word, num_topics, chunksize, alpha, eta, distributed, random_state=None):
```

Comment:
No default values? Are you sure that user **must** pass all of this?

Reply:
The LdaModel defines the default values for these, and the dask worker should use the values that the LdaModel is created. Normally in this case I refactor the config information, then LdaModel would be split into 2, LdaModelConfig configuration, and LdaModelCompute. I see a separate class was created for LdaState, to hold the state of the lda algorithm.

Reply:
`LdaState` used for different propose (this is "inner state", used only for computing).
I understand your point, but in this case, you need to make all parameters optional (or do it in a different way).